hack/verify_pipeline.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -ue
+
+# Use the policies defined in the Enterprise Contract to validate a 
+# pipline definition
+#
+# Tyipcal usage:
+#  oc get pipeline somePipeline -n someNamespace -o jsonpath='{.items[0]}' > pipeline.json
+#  ./verify_pipeline.sh pipeline.json
+
+if [ $# -ge 1 ] && [ -n "$1" ]; then
+  INPUT_FILE=$1
+else
+  echo "The path to a file to test must be specified"
+  exit 1
+fi
+
+# Determine useful directories
+if git rev-parse --is-inside-work-tree >/dev/null 2>&1; then
+  # For hacking/testing
+  ROOT_DIR=$( git rev-parse --show-toplevel )
+fi
+
+SCRIPTS_DIR="$ROOT_DIR/appstudio-utils/util-scripts"
+LIB_DIR="$SCRIPTS_DIR/lib"
+
+# For these you can set the env vars to use non-default values
+EC_WORK_DIR=${EC_WORK_DIR:-/tmp/ecwork}
+
+POLICIES_DIR=${POLICIES_DIR:-"$EC_WORK_DIR/policies"}
+
+mkdir -p $POLICIES_DIR
+
+POLICY_REPO=${POLICY_REPO:-"https://github.com/hacbs-contract/ec-policies.git"}
+POLICY_REPO_REF=${POLICY_REPO_REF:-"main"}
+
+CONFTEST_NAMESPACE=${CONFTEST_NAMESPACE:-"pipeline.main"}
+
+# TODO: If git clone fails, the task does not fail - fix that!
```

Comment:
This comment is old and stale - we have no intention to fix the obsolete bash scripts. Let's remove the incorrect comment.